### PR TITLE
Allow a successful CSV instantiation and running of operator-sdk scorecard

### PR DESCRIPTION
**Pull request by leifmadsen**

### CODE REVIEW

File: deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -16,13 +18,25 @@ metadata:
...
     capabilities: Basic Install
-  name: service-assurance-operator.v0.1.0
-  namespace: sa-telemetry
+    categories: Monitoring
+    description: Service Assurance Framework. Umbrella Operator for instantiating the required dependencies and configuration of various components to build a Service Assurance platform for telco grade monitoring.
+    certified: "false"
+    createdAt: 2019-11-04T08:00:00Z
+    support: Red Hat, Inc.
+    repository: https://github.com/redhat-service-assurance/service-assurance-operator
+    containerImage: quay.io/redhat-service-assurance/service-assurance-operator.0.1.0
```
Comment (csibbitt) [inline deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml]: I'd like to have a real-time conversation with you sometime this week about upstream and downstream for this stuff.

Comment (leifmadsen) [reply deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml]: Sounds like I'm getting called to the principals office.

**Comment by leifmadsen** [comment]
I've expanded the scope of this PR way too much... going to mark this do-not-merge until I can get it into a reasonable form that is eligible for merging.

**Comment by leifmadsen** [comment]
Will need to address somethings so that the scorecard passes fully. There is no reason we don't pass everything since the Operator effectively does nothing :)

```
operator-sdk scorecard
INFO[0000] Using config file: /home/sysop/src/github.com/redhat-service-assurance/service-assurance-operator/.osdk-scorecard.yaml 
basic:
	Spec Block Exists                   : pass
	Labels: 
		"necessity":"required"
		"suite":"basic"

	Status Block Exists                 : pass
	Labels: 
		"necessity":"required"
		"suite":"basic"

	Writing into CRs has an effect      : pass
	Labels: 
		"necessity":"required"
		"suite":"basic"

olm:
	Status fields with descriptors      : fail
	Labels: 
		"necessity":"required"
		"suite":"olm"
	Suggestions: 
		Add a status descriptor for conditions

	Provided APIs have validation       : fail
	Labels: 
		"necessity":"required"
		"suite":"olm"
	Suggestions: 
		Add CRD validation for ServiceAssurance/

	Owned CRDs have resources listed    : pass
	Labels: 
		"suite":"olm"
		"necessity":"required"
	Suggestions: 
		If it would be helpful to an end-user to understand or troubleshoot your CR, consider adding resources [serviceassurances/v1alpha1 replicasets/v1 deployments/v1 configmaps/v1] to the resources section for owned CRD ServiceAssurance

	CRs have at least 1 example         : pass
	Labels: 
		"necessity":"required"
		"suite":"olm"

	Spec fields with descriptors        : pass
	Labels: 
		"necessity":"required"
		"suite":"olm"

```

**Pull request merged**